Disable __init__.py workaround for vendor packages

In preparation for using a shim file, disable the workaround
https://github.com/gazebo-tooling/gz_vendor/pull/26#pullrequestreview-3632406242

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index edb6c84..77731f7 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -1,12 +1,3 @@
-# Append `_configured` to the file name so it doesn't interfere with tests.
-# This happens because pytest will load the `gz.msgs` package from the build directory
-# (because there's an __init__.py file there) instead of being redirected to
-# `gz.msgs11` in the install directory, which is the intent of this `__init__.py` file.
-set(python_init_file ${PROJECT_BINARY_DIR}/python/gz/${GS_DESIGNATION}/__init__.py_configured)
-configure_file(${PROJECT_SOURCE_DIR}/python/src/__init__.py.in ${python_init_file})
-
-install(FILES ${python_init_file} DESTINATION ${GZ_PYTHON_INSTALL_PATH}/gz/${GZ_DESIGNATION}${PROJECT_VERSION_MAJOR} RENAME __init__.py)
-
 if (BUILD_TESTING AND NOT WIN32)
   set(python_tests
     basic_TEST
